test/build_container.sh

```diff
@@ -34,6 +35,11 @@ if [ "$RESTY_IMAGE_TAG" == "stretch" ]; then
   sed -i 's/unzip git/unzip git gdebi/g' docker-kong/ubuntu/Dockerfile
 fi
 
+if [ "$RESTY_IMAGE_BASE" == "debian"] && [ "$RESTY_IMAGE_TAG" == "9" ]; then
+  sed -i 's/apt install --yes /gdebi -n /g' docker-kong/ubuntu/Dockerfile
+  sed -i 's/unzip git/unzip git gdebi/g' docker-kong/ubuntu/Dockerfile
+fi
+
```

Comment:
Shouldn't this just have been an update the above `stretch` conditional?

Reply:
yeah I think that's what I meant to do ... but failed. I dunno will fix lol

Reply:
nope backwards compatible makes complete sense; I was referring to exactly what you were "supposing"

Reply:
> if [ "$RESTY_IMAGE_TAG" == "stretch" ] || [ "$RESTY_IMAGE_TAG" == "9" ]; then

We should probably future proof this as there is a  RHEL 9 on the horizon ([a beta was just released](https://www.redhat.com/en/blog/whats-new-rhel-90-beta)) and a `RESTY_IMAGE_TAG=9` there would apply to the conditional above. For complete backwards compatibility we could do this and remove the addition in this PR:

```
if [ "$RESTY_IMAGE_TAG" == "stretch" ] ||
  { [ "$RESTY_IMAGE_BASE" == "debian" ] && [ "$RESTY_IMAGE_TAG" == "9" ]; } then
  sed -i 's/apt install --yes /gdebi -n /g' docker-kong/ubuntu/Dockerfile
  sed -i 's/unzip git/unzip git gdebi/g' docker-kong/ubuntu/Dockerfile
fi
```

Reply:
This hax is only necessary for earlier than stretch so when stretch is EOL is should go away

Reply:
But we also need `jessie` (8) which neither are ETLS yet:

- Debian 8 (jessie) ELTS is 2022/06/30
- Debian 9 (stretch) ETLS hasn't been defined yet

see see https://wiki.debian.org/LTS/Extended for more details.